lib/v2/screens/transfer/send_confirmation/interactor/send_confirmation_bloc.dart

```diff
@@ -9,20 +9,18 @@ import 'package:seeds/v2/utils/cap_utils.dart';
 
 /// --- BLOC
 class SendConfirmationBloc extends Bloc<SendConfirmationEvent, SendConfirmationState> {
-  SendConfirmationBloc() : super(SendConfirmationState.initial());
+  SendConfirmationBloc(arguments) : super(SendConfirmationState.initial(arguments));
```

Comment:
this is how we pass initial arguments to the state. 
These are usually arguments passed by the view on initial page load